exercises/wordy/example.js

```diff
@@ -1,47 +1,31 @@
-class ArgumentError extends Error {
-  constructor() {
-    super();
-    this.name = 'argument error';
-    this.message = 'oops';
-  }
-}
-
-const re = new RegExp(/(plus|minus|divided by|multiplied by)+/g);
+export const answer = question => {
+  question = question
+    .replace(/plus/g, ')+')
+    .replace(/minus/g, ')-')
+    .replace(/multiplied by/g, ')*')
+    .replace(/divided by/g, ')/')
+    .replace('What is', '')
+    .replace(/\?/g, '')
+    .trim();
 
-class Wordy {
-  constructor(question) {
-    this.numbers = question.match(/[-]{0,1}\d+/g);
-    this.operands = question.match(re);
-  }
+  const closingParenthesisCount = (question.match(/\)/g) || []).length;
+  const openingParenthesis = '('.repeat(closingParenthesisCount);
 
-  answer() {
-    if (!this.numbers || !this.operands) {
-      throw new ArgumentError();
-    }
-    let result = +this.numbers[0];
+  question = openingParenthesis + question;
 
-    this.operands.forEach((operand, index) => {
-      const nextNumber = +this.numbers[index + 1] || null;
+  if(/^[0-9]+$/.test(question)) {
+    return parseInt(question);
+  } else if (question.length === 0) {
+    throw new Error('Syntax error')
+  }
 
-      switch (operand) {
-        case 'plus':
-          result += nextNumber;
-          break;
-        case 'minus':
-          result -= nextNumber;
-          break;
-        case 'multiplied by':
-          result *= nextNumber;
-          break;
-        case 'divided by':
-          result /= nextNumber;
-          break;
-        default:
-          break;
-      }
-    });
-    return result;
+  if (!/[+/*-]+/.test(question)) {
+    throw new Error('Unknown operation');
   }
-}
 
-export { Wordy as WordProblem, ArgumentError };
+  try {
+    return eval(question);
```

Comment:
Whilst the example file is never meant to be an optimal solution, I am not sure if we should be having `eval` in here -- for the simple reason that people might copy and use it and it will start discussions not meant for this exercise.